Create new "previous-root" key purpose for root key migrations (#2639)
hashicorp/boundary: website/content/docs/configuration/kms/index.mdx

# `kms` Stanza

The `kms` stanza configures Boundary kms-specific parameters. Multiple stanzas
of the same `purpose` are merged, with the last one specified being used for

> encryption, and all specified being available for decryption.

Reviewer: ```suggestion
encryption. All specified stanzas are available for decryption.
```
